store/sql_post_store.go

```diff
@@ -642,6 +642,7 @@ func (s SqlPostStore) Search(teamId string, userId string, params *model.SearchP
 				Posts
 			WHERE
 				DeleteAt = 0
+				AND Type NOT LIKE '` + model.POST_SYSTEM_MESSAGE_PREFIX + `%'
```

Comment:
does this need to be a like clause?  I think it's going to be very slow.  We might want to load it in a query analyzer and see.